### ISSUE TITLE
pip install does not work in a fresh conda environment

### ISSUE THREAD

**Issue by hejung** [bio-phys/MDBenchmark#176]
**Expected behavior**

The following should result in a working installation:
```bash
conda create -n test_mdb python
conda activate test_mdb
pip install mdbenchmark
```

**Actual behavior**
```bash
mdbenchmark --help
```
results in the following traceback:
```bash
Traceback (most recent call last):
  File "/u/hejung/conda-envs/test_mdb/bin/mdbenchmark", line 5, in <module>
    from mdbenchmark import cli
  File "/u/hejung/conda-envs/test_mdb/lib/python3.10/site-packages/mdbenchmark/__init__.py", line 21, in <module>
    from mdbenchmark.cli import cli  # noqa: F401
  File "/u/hejung/conda-envs/test_mdb/lib/python3.10/site-packages/mdbenchmark/cli/__init__.py", line 20, in <module>
    from mdbenchmark.cli.commands import cli  # noqa: F401
  File "/u/hejung/conda-envs/test_mdb/lib/python3.10/site-packages/mdbenchmark/cli/commands.py", line 24, in <module>
    from mdbenchmark.cli.validators import (
  File "/u/hejung/conda-envs/test_mdb/lib/python3.10/site-packages/mdbenchmark/cli/validators.py", line 23, in <module>
    from mdbenchmark import console, utils
  File "/u/hejung/conda-envs/test_mdb/lib/python3.10/site-packages/mdbenchmark/utils.py", line 28, in <module>
    from jinja2 import ChoiceLoader, Environment, FileSystemLoader, PackageLoader
  File "/u/hejung/conda-envs/test_mdb/lib/python3.10/site-packages/jinja2/__init__.py", line 12, in <module>
    from .environment import Environment
  File "/u/hejung/conda-envs/test_mdb/lib/python3.10/site-packages/jinja2/environment.py", line 25, in <module>
    from .defaults import BLOCK_END_STRING
  File "/u/hejung/conda-envs/test_mdb/lib/python3.10/site-packages/jinja2/defaults.py", line 3, in <module>
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
  File "/u/hejung/conda-envs/test_mdb/lib/python3.10/site-packages/jinja2/filters.py", line 13, in <module>
    from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/u/hejung/conda-envs/test_mdb/lib/python3.10/site-packages/markupsafe/__init__.py)
```

**Your version of MDBenchmark:**
running `mdbenchmark --version` fails with the same as above.
pip list gives `mdbenchmark        3.0.1`

**Comment by ksebaz** [comment]
This is an internal error in an unsupported version of jinja2. It happens here because jinja2 is pinned to version < 3. Sticking to that, MarkupSafe must be pinned to some compatible version, i.e., 2.0.1. Alternatively, jinja2 could be updated to a supported release.

**Comment by ksebaz** [comment]
This issue should be resolved with #184.
